### DISCUSSION TITLE
Salt-Cloud provider for Digital Ocean requires a way to specify the VPC to use

### DISCUSSION THREAD

**Discussion by Office-Klerk** [saltstack/salt#67548] [Feature Requests]
The Salt-Cloud provider for Digital Ocean (DO) allows you to connect the new VM/Droplet to private networking using DO's VPC functionality but doesn't allow you to specify the VPC you want to connect to, it only uses the default VPC as specified in your DO account.  Unfortunately, DO also don't allow you to change the default.

A sample provider.conf file would be along the lines of:

```
digital-ocean:
   private_networking: true
   vpc:  my-vpc-name
```

which would translate to adding the vpc_uuid parameter in the API call to DO.

**Comment by sagetherage** [comment]
@waynew this is good one for Test Clinic since it has a PR that needs a test case. Assigning you only for that reason and won't put into a committed release until tests are written.
